amivapi/validation.py

```diff
@@ -292,8 +292,9 @@ def _validate_aspect_ratio(self, aspect_ratio, field, value):
         """
         width, height = aspect_ratio
         error = False
-        img = Image.open(value.stream)
-        print(img.size, aspect_ratio)
+        # Load file (and reset stream so it can be saved correctly afterwards)
+        img = Image.open(value)
```

Comment:
Note: `value` is a file object and can be directly read from, there is no need to access `value.stream` explicitly, thats why I have removed it.